.github/workflows/publish.yaml

```diff
@@ -57,44 +57,44 @@ jobs:
       name: 'building lexi-api'
       uses: 'gradle/gradle-build-action@v2'
       with:
+        build-root-directory: './lexi'
         cache-disabled: 'true'
```

Comment:
The only change in the YAML is this change of order of `build-root-directory` parameter, apparently the order got changed somewhere between the two versions, and now should be stable.